ols/src/query_helpers/question_validator.py

```diff
@@ -51,9 +58,7 @@ def validate_question(
 
         logger.info(f"{conversation} Validating query")
 
-        bare_llm = LLMLoader(
-            self.provider, self.model, params={"min_new_tokens": 1, "max_new_tokens": 4}
-        ).llm
+        bare_llm = self.get_llm()
```

Comment:
summarizing my other comments:  i would rather see this function be provided by llmloader and take provider/model/params arguments so that:

1) it's obvious at point of use what model/provider/params are going to be used
2) it's possible for a query helper to get refs to multiple llms or multiple instances of one llm w/ different params
3) it's easier to change the parameters dynamically